src/test/scala/akka/persistence/jdbc/query/CurrentEventsByTagTest.scala

```diff
@@ -221,4 +222,4 @@ class OracleScalaCurrentEventsByTagTest extends CurrentEventsByTagTest("oracle-s
 
 class SqlServerScalaCurrentEventsByTagTest extends CurrentEventsByTagTest("sqlserver-shared-db-application.conf") with SqlServerCleaner
 
-class H2ScalaCurrentEventsByTagTest extends CurrentEventsByTagTest("h2-shared-db-application.conf") with H2Cleaner
+class H2ScalaCurrentEventsByTagTest extends CurrentEventsByTagTest("h2-shared-db-application.conf", 5000) with H2Cleaner
```

Comment:
Instead of changing the test to make it green. Did you try experimenting with different settings of the jdbc-journal.batchSize?